docs/reference/modules/indices/query_cache.asciidoc

```diff
@@ -5,6 +5,7 @@ The query cache is responsible for caching the results of queries.
 There is one queries cache per node that is shared by all shards.
 The cache implements an LRU eviction policy: when a cache becomes full, the
 least recently used data is evicted to make way for new data.
+(It is not possible to look at the contents being cached.)
```

Comment:
why is the new sentence added within parentheses? That looks odd to me, maybe the parentheses are not needed.

Reply:
I wasn't sure what the best way to word it was, or if we have this on the roadmap.
But yes, parantheses are not necessarily needed.

Reply:
Can you remove them please? :)